sky/utils/controller_utils.py

```diff
@@ -32,6 +32,7 @@
 from sky.utils import ux_utils
 
 if typing.TYPE_CHECKING:
+    from sky import dag as dag_lib
```

Comment:
why add this?

Reply:
I added a function which uses dag_lib to this file and later I removed that function and forgot to remove this import.